Dockerfile

```
@@ -1,4 +1,4 @@
-FROM quay.io/redhat-developer/go-toolset:builder-golang-1.18 as builder
+FROM golang:1.20 as builder
```

```suggestion
FROM quay.io/redhat-developer/go-toolset:builder-golang-1.20 as builder
```
@sadlerap You can copy the `golang:1.20` base image to quay with `skopeo copy docker://golang:1.20 docker://quay.io/redhat-developer/go-toolset:builder-golang-1.20` as we did before to avoid depenency on `docker.io` registry, which caused SBO troubles with rate limits in the past. Once you copy the image to quay you can then keep the base image `quay.io/redhat-developer/go-toolset:builder-golang-1.20`

---

I have pushed the `quay.io/redhat-developer/go-toolset:builder-golang-1.20` image to quay.